cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+{{ $version := "master-101" }}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fabric-gateway-controller
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: fabric-gateway-controller
+    version: {{ $version }}
+spec:
+{{ if eq .ConfigItems.fabric_gateway_controller_mode "disabled" }}
+  replicas: 0
+{{ else }}
+  replicas: 1
+{{ end }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      deployment: fabric-gateway-controller
+  template:
+    metadata:
+      labels:
+        deployment: fabric-gateway-controller
+        application: kubernetes
+        component: fabric-gateway-controller
+        version: {{ $version }}
+      annotations:
+        logging/destination: "{{ .Cluster.ConfigItems.log_destination_infra }}"
+        prometheus.io/path: /metrics
+        prometheus.io/port: "7979"
+        prometheus.io/scrape: "true"
+    spec:
+      serviceAccountName: fabric-gateway-controller
```

Comment:
Do we need
```yaml
      priorityClassName: "{{ .Cluster.ConfigItems.system_priority_class }}"
```
like in other controllers?

Reply:
yes

Reply:
Added